doc/radosgw/admin.rst

```diff
@@ -197,41 +197,45 @@ in a command of the following form:
 Remove a User
 -------------
 
-When you remove a user, the user and subuser are removed from the system.
-However, you may remove just the subuser if you wish. To remove a user (and
-subuser), specify ``user rm`` and the user ID. ::
+The act of removing a user removes both the user and any subusers associated
```

Comment:
I think we should try again with this para.  The original has "you", which I guess you don't want?  But the second makes the subject to be "the act of removing" which is not natural, and then tells the user  to "remember" something that it is just now describing, which doesn't make sense.

Reply:
@mattbenjamin: I appreciate your review of this. I have no objection to the use of second-person (the use of "you" in the original text) in instruction of this kind. I just thought that it might be clearer the way it was written here. Your fresh eyes let me know that it wasn't. So: I've reworked it, and made it available for review here: https://github.com/ceph/ceph/pull/55030

Let me know what you think.